client/src/helpers/api.js

```diff
@@ -8,7 +8,9 @@ const instance = axios.create({
 export const getStatus = () => {
   return instance.get("/api/status").then(res => res);
 };
-
+export const getCourses = () => {
+  return instance.get("/api/courses").then(res => res);
```

Comment:
Hmm - I don't think the `.then(res => res);` after the promise helps with anything.